data/README.md

```diff
@@ -4,6 +4,27 @@ A simple implementation of data generation of [Improve Mathematical Reasoning in
 
 ## Generating data
 
+Edit the `config.yaml` file to set the model and other parameters. Input json file should be a list of problems and answers with key `problem` and `final_answer`.
+
+```yaml
+input:
+  json_file_path: 'extracted_problems_and_answers.json'
+
+output:
+  log_file_path: 'processing_log.log'
+
+processing:
+  initial_rollouts: 20
+  num_rollouts: 5
+
+model:
+  model_type: "hf"  # Supported types: "hf", "openai", "anthropic"
+  model_name: "Qwen/Qwen2.5-Math-7B-Instruct" # model name, e.g. "Qwen/Qwen2.5-Math-7B-Instruct", "gpt-4o"
+  model_args:
+    max_tokens: 512
```

Comment:
is `max_tokens` 512 sufficiently large?

Reply:
Hi, thank you for your contribution! We recently reimplemented [OmegaPRM](https://github.com/openreasoner/openr/tree/main/data/omegaPRM_v2), and I believe your code for supporting API models could integrate smoothly with our current setup. Let us know if you'd like to collaborate further on this! 
Thanks again!

Reply:
> is `max_tokens` 512 sufficiently large?

A `max_tokens` setting of 512 might not be enough for complex or long responses, but we can change this setting in the config to fit different needs.